include/baresip.h

```diff
@@ -396,6 +401,7 @@ struct config_avt {
 	bool rtp_stats;         /**< Enable RTP statistics          */
 	uint32_t rtp_timeout;   /**< RTP Timeout in seconds (0=off) */
 	bool bundle;            /**< Media Multiplexing (BUNDLE)    */
+	enum receive_mode rxmode;       /**< RTP RX processing mode */
```

Comment:
is it possible to use one method for audio, and a different mode for video ?

Reply:
Currently not. Same mode for audio and video.
This is simpler. And maybe this helps to make audio+video lip sync later.

Reply:
I have to correct myself. For now I only worked on thread safety for audio processing. I think for now the RX thread for video has to be turned off.